utils/src/main/java/de/hellfish/ngrok/utils/MyHttpRequest.java

```diff
@@ -0,0 +1,37 @@
+package de.hellfish.ngrok.utils;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Map;
+
+@Getter
+@ToString
+@EqualsAndHashCode
+public final class MyHttpRequest {
```

Comment:
Name?

Reply:
I changed name cause it duplicated one with java.net.http.HttpRequest and they were in the same blocks of code

Reply:
"My" prefix is meaningless.